File: front-end/src/app/service-models/app/v1/client/client-excel-file-summary-s-m.ts
import { ClientEmployeeAttendanceExtendedUserSM } from './client-employee-attendance-extended-user-s-m';

export class ClientExcelFileSummarySM {
    attendanceSummary!: Array<ClientEmployeeAttendanceExtendedUserSM>;
    fromDate!: string;
    toDate!: string;
    totalRecordsCount!: number;
    numberOfRecordsNotAdded!: number;
    employeeRecordsAddedCount!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/ClientExcelFileSummarySM.cs
using System.Collections.Generic;

namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class ClientExcelFileSummarySM
    {
        public List<ClientEmployeeAttendanceExtendedUserSM> AttendanceSummary { get; set; }
        public string FromDate { get; set; }
        public string ToDate { get; set; }
        public int TotalRecordsCount { get; set; }
        public int NumberOfRecordsNotAdded { get; set; }
        public int EmployeeRecordsAddedCount { get; set; }

    }
}

